isis/src/qisis/apps/ipce/IpceMainWindow.cpp

```diff
@@ -80,24 +80,17 @@ namespace Isis {
       QMainWindow(parent) {
     m_maxThreadCount = -1;
 
-    //  Set the initialize size of the mainwindow to fullscreen so that created views do not
-    //  get squished.  Saved projects with view had the internal widgets squished because the
-    //  initial size of this mainwindow was small and it does not get restored to the saved project
-    //  size until after views are created.  For instance, the viewports within a CubeDnView were
-    //  restored to a small size based on the original mainwindow size.  If the internal state
-    //  of the views such as the viewport sizes, zooms, etc get serialized, this code will not be
-    //  needed.
-    QDesktopWidget deskTop;
-    QRect mainScreenSize = deskTop.availableGeometry(deskTop.primaryScreen());
-    resize(mainScreenSize.width(), mainScreenSize.height());
-
     QWidget *centralWidget = new QWidget;
     setCentralWidget(centralWidget);
+
+    setTabPosition(Qt::LeftDockWidgetArea, QTabWidget::South);
+    //setDockOptions(GroupedDragging | AllowTabbedDocks);
```

Comment:
Commented-out line: doe sit need to go?

Reply:
I was going to leave this here because I reference it in my history entry, but if we can remove it, I'm fine with that.